idaes/power_generation/flowsheets/NGFC/NGFC_flowsheet.py

```diff
@@ -0,0 +1,1445 @@
+    m.fs.reformer_recuperator.tube_inlet.mole_frac_comp[0, 'C4H10'].fix(0.004)
+    m.fs.reformer_recuperator.tube_inlet.mole_frac_comp[0, 'CO'].fix(0)
+    m.fs.reformer_recuperator.tube_inlet.mole_frac_comp[0, 'CO2'].fix(0.01)
+    m.fs.reformer_recuperator.tube_inlet.mole_frac_comp[0, 'H2'].fix(0)
+    m.fs.reformer_recuperator.tube_inlet.mole_frac_comp[0, 'H2O'].fix(0)
+    m.fs.reformer_recuperator.tube_inlet.mole_frac_comp[0, 'N2'].fix(0.016)
+    m.fs.reformer_recuperator.tube_inlet.mole_frac_comp[0, 'O2'].fix(0)
+    m.fs.reformer_recuperator.tube_inlet.mole_frac_comp[0, 'Ar'].fix(0)
+
+    # recuperator conditions
+    m.fs.reformer_recuperator.tube_outlet.temperature.fix(1010.93)
+    m.fs.reformer_recuperator.overall_heat_transfer_coefficient.fix(80)
+
+    # natural gas expander conditions
+    m.fs.NG_expander.outlet.pressure.fix(206843)  # equal to 30 psia
+    m.fs.NG_expander.efficiency_isentropic.fix(0.9)
+
+    # internal reformation percentage
+    m.fs.reformer_bypass.split_fraction[0, 'bypass_outlet'].fix(0.6)
+
+    # air to reformer
+    m.fs.air_compressor_s1.inlet.flow_mol[0] == 1332.9  # mol/s
+    m.fs.air_compressor_s1.inlet.temperature.fix(288.15)  # K
+    m.fs.air_compressor_s1.inlet.pressure.fix(101353)  # Pa, equal to 14.7 psia
+    m.fs.air_compressor_s1.inlet.mole_frac_comp.fix(0)
+    m.fs.air_compressor_s1.inlet.mole_frac_comp[0, 'CO2'].fix(0.0003)
+    m.fs.air_compressor_s1.inlet.mole_frac_comp[0, 'H2O'].fix(0.0104)
+    m.fs.air_compressor_s1.inlet.mole_frac_comp[0, 'N2'].fix(0.7722)
+    m.fs.air_compressor_s1.inlet.mole_frac_comp[0, 'O2'].fix(0.2077)
+    m.fs.air_compressor_s1.inlet.mole_frac_comp[0, 'Ar'].fix(0.0094)
+
+    # air compressors and intercoolers
+    m.fs.air_compressor_s1.outlet.pressure.fix(144790)  # Pa, equal to 21 psia
+    m.fs.air_compressor_s1.efficiency_isentropic.fix(0.84)
+
+    m.fs.intercooler_s1.outlet.temperature.fix(310.93)  # K, equal to 100 F
+    m.fs.intercooler_s1.deltaP.fix(-3447)  # Pa, equal to -0.5 psi
+
+    m.fs.air_compressor_s2.outlet.pressure.fix(206843)  # Pa, equal to 30 psia
+    m.fs.air_compressor_s2.efficiency_isentropic.fix(0.84)
+
+    m.fs.intercooler_s2.outlet.temperature.fix(310.93)  # K, equal to 100 F
+    m.fs.intercooler_s2.deltaP.fix(-3447)  # Pa, equal to -0.5 psi
+
+    # steam to reformer
+    m.fs.reformer_mix.steam_inlet.flow_mol[0] == 464.77  # mol/s
+    m.fs.reformer_mix.steam_inlet.temperature.fix(422)  # K
+    m.fs.reformer_mix.steam_inlet.pressure.fix(206843)  # Pa, equal to 30 psia
+    m.fs.reformer_mix.steam_inlet.mole_frac_comp.fix(0)
+    m.fs.reformer_mix.steam_inlet.mole_frac_comp[0, 'H2O'].fix(1)
+
+    # reformer outlet pressure
+    m.fs.reformer.outlet.pressure.fix(137895)  # Pa, equal to 20 Psi
+    m.fs.reformer.outlet.temperature.fix(1060.93)  # K, equal to 1450 F
+
+
+def scale_flowsheet(m):
+    # set NG_props default scaling
+    m.fs.NG_props.set_default_scaling("flow_mol", 1e-3)
```

Comment:
A comment for the near future- generic property packages will support the scaling tools in the next few days.